x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml

```diff
@@ -163,4 +163,4 @@ setup:
   - match: {esql.functions.cos: $functions_cos}
   - gt: {esql.functions.to_long: $functions_to_long}
   - match: {esql.functions.coalesce: $functions_coalesce}
-  - length: {esql.functions: 118} # check the "sister" test above for a likely update to the same esql.functions length check
+  - length: {esql.functions: 119} # check the "sister" test above for a likely update to the same esql.functions length check
```

Comment:
Shouldn't [the other test](https://github.com/elastic/elasticsearch/blob/5c14a80d1770bbea56a5ee6e0d5b7982d5961798/x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/60_usage.yml#L95) be changed as well?

Also, I am not clear about the "promotion" of `categorize`. Looking at [EsqlCapabilities](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java#L410) I see it's still a snapshot only feature test-wise.

Reply:
Also, if this is about snapshot/non-snapshot thingies, shouldn't the PR run with `test-release` label?
Even the issue it's trying to fix (https://github.com/elastic/elasticsearch/issues/117862) comes from test-release CI runs.

Reply:
> I see it's still a snapshot only feature test-wise.
It's going to be un-snapshotted soon!

Thanks for pointing out the `test-release` label, I was unaware of that one.

Reply:
The other test is good because the function existed already (but only in snapshot).

The promotion happened in [EsqlFunctionRegistry](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/EsqlFunctionRegistry.java#L270), I think the capabilities should be changed as well. Since we are on it, let me do it now

Reply:
Done

Reply:
Yeah, the un-snapshotting of the capability in https://github.com/elastic/elasticsearch/pull/117881 should make sure that the CATEGORIZE test also run on release tests. Thanks for looking into this Luigi!